zuul.d/adoption.yaml

```diff
@@ -152,11 +152,10 @@
                 ip: 172.19.0.100
                 config_nm: false
 
-# TODO(marios) remove this definition after new OSP17 job is running and stable
 - job:
-    name: cifmw-data-plane-adoption-github-rdo-centos-9-extracted-crc
-    parent: content-provider-data-plane-adoption-github-rdo-centos-9-extracted-crc
-    files: &dpa_job_files
+    name: cifmw-data-plane-adoption-OSP-17-to-extracted-crc
+    parent: content-provider-data-plane-adoption-OSP-17-to-extracted-crc
```

Comment:
Is not the firstime I hear complains (and I complain too) about using uppercase names here. Is really necessary?

Reply:
I have no strong opinion either way, but if we want to change it I would do it in a dedicated PR and duplicating it first, since there are other jobs that inherit from it, wdyt @marios?

Reply:
ack ... cleanup is definitely on the agenda for this quarter, including the job names and removing unnecessary duplicate definition (with only slightly different names)